src/earthengine/ee_worker_utils.js

```diff
@@ -116,3 +124,49 @@ export const getClassifiedImage = (eeImage, { legend = [], params }) => {
 
     return { eeImage: zones, params: { min, max, palette } }
 }
+
+// Apply filter to image collection
+export const applyFilter = (collection, filter = []) => {
+    let filtered = collection
+
+    filter.forEach(f => {
+        if (ee.Filter[f.type]) {
+            filtered = filtered.filter(
+                ee.Filter[f.type].apply(this, f.arguments)
+            )
+        }
+    })
+
+    return filtered
+}
+
+// Apply methods to image cells
+export const applyMethods = (eeImage, methods = []) => {
+    let image = eeImage
+
+    if (Array.isArray(methods)) {
+        methods.forEach(m => {
+            if (image[m.name]) {
+                image = image[m.name].apply(image, m.arguments)
+            }
+        })
+    } else {
+        // Backward compatibility for format used before 2.40
+        Object.keys(methods).forEach(m => {
+            if (image[m]) {
+                image = image[m].apply(image, methods[m])
+            }
+        })
+    }
+
+    return image
+}
+
+// Mask out clouds from satellite images
+export const applyCloudMask = (collection, cloudScore) => {
+    const { datasetId, band, clearTreshold } = cloudScore
```

Comment:
clearTreshold => clearThreshold (missing an `h`)

Reply:
@BRaimbault @turban I need to understand where "clearTreshold" is coming from. Is that from the google or is it something we've named ourselves? Because "threshold" is spelled wrong.

Reply:
Changed to "clearThreshold". 

It will be used to remove clouds from satellite imagery: 
https://github.com/dhis2/maps-app/blob/f8b0b60e4412f4573418664a8a4df252c0ba6fc3/src/constants/earthEngineLayers/satellite_imagery_Sentinel-2.js#L23

This will be used at a later stage in DHIS2 Maps.